internal/domain/release.go

```diff
@@ -423,58 +425,72 @@ func (r *Release) downloadTorrentFile(ctx context.Context) error {
 			// Continue processing the response
 		case http.StatusMovedPermanently, http.StatusFound, http.StatusSeeOther, http.StatusTemporaryRedirect, http.StatusPermanentRedirect:
 			// Handle redirect
-			return retry.Unrecoverable(errors.New("redirect encountered for torrent (%v) file (%v) from '%v' - status code: %d. Check indexer keys.", r.TorrentName, r.TorrentURL, r.Indexer, resp.StatusCode))
+			return retry.Unrecoverable(errors.New("redirect encountered for torrent (%v) file (%v) - status code: %d - check indexer keys for %s", r.TorrentName, r.TorrentURL, resp.StatusCode, r.Indexer))
```

Comment:
Redirects are a great way to see a user visible error, but they're also still valid for a tracker announcing on one url during a domain flip. These events are pretty rare in comparison, but just wanted to mention it.

Reply:
That's a valid point. Then we should probably not return err on `StatusMovedPermanently, StatusTemporaryRedirect, StatusPermanentRedirect`?

Reply:
I don't think so, we should parse the results still but log that it was a redirect. I don't know what happens when we tell the go httpclient to follow redirects.

Reply:
It will follow (10) redirects by default, so that should be fine.